dashboard/modules/job/job_manager.py

```diff
@@ -73,11 +76,17 @@ def get_last_n_log_lines(
     ) -> str:
         log_tail_iter = self.tail_logs(job_id)
         log_tail_deque = deque(maxlen=num_log_lines)
+        line_length = 0
```

Comment:
can we name this something like `total_length` instead of `line_length` give that it is the total length of the message across lines??